docs/intro/tutorial08.txt

```diff
@@ -48,12 +48,16 @@ The steps are not duplicated in this tutorial, because as a third-party
 package, it may change separately to Django's schedule.
 
 Once installed, you should be able to see the DjDT "handle" on the right side
-of the browser window when you refresh the polls application. Click it to open
+of the browser window when you refresh the admin of your project. Click it to open
```

Comment:
```suggestion
of the browser window when you browse to the admin of your project (http://localhost:8000/admin). Click it to open
```
I'm not quite sure the user will be in the polls area of their web application. It might be more appropriate to tell them to browse to the admin. I'm not sure about including the URL.

Reply:
👌 https://github.com/django/django/pull/18285/commits/4e774ca18c9ae93eece57907c85567c14454f7bb